src/app.js

```diff
@@ -155,7 +149,7 @@ export default function (app) {
 
       for (var name in node.data) {
         if (name === "onCreate") {
-          defer(node.data[name], element)
+          node.data[name](element)
```

Comment:
@zaceno Ah, that explains why I originally was deferring these stuff! Well observed 👍